### PR TITLE
feat: page banner component

### LINKED ISSUE #520: Add travel mode landing page banner to the Banner patterns page (including documentation)
Screenshot 2020-12-03 at 15 16 56

#### What does it do?
- Helps users to identify the travel mode service
- Provides users with a short description of the travel service in the West Midlands

#### When to use it?
- Only on Travel Mode Landing Pages

#### When not to use it?
- On pages where the mode of travel is not operated by West Midlands Network, such as E-scooters

#### How it works
- The image container width will expand if users are viewing the page with a browser width higher than 1280px. 
- The image within the container will zoom to fill the size of the image container.
- The image will be hidden on mobile devices. Our previous research found mobile users do not see the value of images on mobile pages.
- The image, modal icon and the West Midlands Network logo variant will change for each travel mode landing page

### PR DESCRIPTION
**Pull request by houbly**
- Added new page banner component (wmndsPageBanner)  with responsive travel mode page banner
- Added component props and examples
- Converted banner pattern page to markdown

fixes #520

### CODE REVIEW

**Review by jkmnch — COMMENTED** [review]
Ideally, this new banner pattern should go under the Banner section of the Patterns page. Is there a reason why this was changed?

**Review by mrmjprice — CHANGES_REQUESTED** [review]
## Mode values:
- [ ] Add cycling
- [x] Add walking
- [ ] Add none

If mode not specified, default to primary colour with no icon.

**Comment by houbly** [comment]
> Ideally, this new banner pattern should go under the Banner section of the Patterns page. Is there a reason why this was changed?

My bad. I don't think it was specified and it seemed like a separate component to banner. I've moved it to the banner page now.

**Review by jkmnch — APPROVED** [review]

**Review by mrmjprice — APPROVED** [review]

File: src/wmnds/patterns/page-banner/_example.njk [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+{% from "wmnds/components/page-banner/_page-banner.njk" import wmndsTravelModeBanner %}
+
+{{ 
+  wmndsPageBanner({
+    title: "Buses in the West Midlands",
+    contentHTML: "<p>It’s easy to travel by bus in the West Midlands. Buses are frequent, with many routes operating late into the evening and on Sundays.</p>",
+    image: "/img/component-images/banner/bus-banner-img.png",
+    mode: "bus",
+    logo: "/img/logos/WM Bus/Landscape/White.png",
+    logoAltText: "West Midlands Bus"
```
Comment (daylesalmon) [inline src/wmnds/patterns/page-banner/_example.njk:11]: Going to remove these two fields and put them within the pattern.

**Review by daylesalmon — CHANGES_REQUESTED** [review]

**Review by daylesalmon — APPROVED** [review]

**Pull request merged**